packagingbuild/focal/Dockerfile

```diff
@@ -59,7 +55,7 @@ RUN curl https://bootstrap.pypa.io/get-pip.py | python3.8 - virtualenv==20.4.0 p
 # Use upstream dh-virtualenv master branch that includes StackStorm's 'python' shebang fix since 8 Dec 2020.
 # We reset the repository to use the known good commit sha id 2dc93574865d6dd9e1fc470541e3232dcbad3337
 RUN apt-get -y install \
-        python3-virtualenv python3-setuptools python3-mock python3-sphinx dh-exec dh-python libjs-jquery libjs-underscore python3-sphinx-rtd-theme && \
+        python3-setuptools python3-mock python3-sphinx dh-exec dh-python libjs-jquery libjs-underscore python3-sphinx-rtd-theme && \
         apt-get clean && \
         git clone --branch master https://github.com/spotify/dh-virtualenv.git /tmp/dh-virtualenv && \
         cd /tmp/dh-virtualenv && \
```

Comment:
The error message appears to be:
```
Preparing to unpack .../dh-virtualenv_1.2.2-1_all.deb ...
Unpacking dh-virtualenv (1.2.2-1) ...
[91mdpkg: dependency problems prevent configuration of dh-virtualenv:
dh-virtualenv depends on virtualenv | python3-virtualenv (>= 1.7) | python3.8-venv; however:
Package virtualenv is not installed.
Package python3-virtualenv is not installed.
Package python3.8-venv is not installed.
dpkg: error processing package dh-virtualenv (--install):
dependency problems - leaving unconfigured
```

Maybe it's possible to go with the `python3.8-venv` as a dependency instead of conflicting `python3-virtualenv`?

Reply:
Odd, I didn't get that error when I ran the build process locally.  I'll take a look.

Reply:
Turns out I was on the wrong branch.  :facepalm:   I'm testing the package build with `python3.8-venv` installed rather than `python3-virtualenv`.  If it still conflicts with the pip installed `virtualenv` I think we'll have to revert to the workaround I proposed originally in the `st2-packages` repo.

Reply:
`python3.8-venv` doesn't appear to conflict with pip installed `virtualenv`.

Reply:
That's good news :+1: